<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploaderWindow</class>
 <widget class="QMainWindow" name="UploaderWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>279</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>File Uploader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="uploaderTab">
         <attribute name="title">
          <string>Upload</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QGridLayout" name="uploaderLayout">
            <item row="1" column="0">
             <widget class="QPushButton" name="uploadButton">
              <property name="text">
               <string>Upload</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QProgressBar" name="uploadProgressBar">
              <property name="toolTip">
               <string/>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="uploadStatusEdit"/>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="filePickerLayout">
              <item>
               <widget class="QLabel" name="filePickerLabel">
                <property name="text">
                 <string>No file chosen...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="filePickerButton">
                <property name="text">
                 <string>Choose file...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="downloaderTab">
         <attribute name="title">
          <string>Download</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="downloaderLayout">
            <item>
             <widget class="QLineEdit" name="downloadPickerEdit">
              <property name="placeholderText">
               <string>paste token here</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="downloadButton">
              <property name="text">
               <string>Download</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="downloadProgressBar">
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="logTab">
         <attribute name="title">
          <string>Log</string>
         </attribute>
         <widget class="QTextBrowser" name="logBrowser">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>591</width>
            <height>471</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>UploaderWindow</receiver>
   <slot>performDownload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>performDownload()</slot>
 </slots>
</ui>
